<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CourseEditor</class>
 <widget class="QWidget" name="CourseEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>491</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="KMessageWidget" name="m_messageWidget"/>
   </item>
   <item>
    <layout class="QFormLayout" name="m_formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::ExpandingFieldsGrow</enum>
     </property>
     <property name="horizontalSpacing">
      <number>4</number>
     </property>
     <property name="verticalSpacing">
      <number>4</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="m_titleLabel">
       <property name="text">
        <string>Title:</string>
       </property>
       <property name="buddy">
        <cstring>m_titleLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="KLineEdit" name="m_titleLineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="m_keyboardLayoutLabel">
       <property name="text">
        <string>Keyboard layout:</string>
       </property>
       <property name="buddy">
        <cstring>m_keyboardLayoutComboBox</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="KeyboardLayoutComboBox" name="m_keyboardLayoutComboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="m_descriptionLabel">
       <property name="text">
        <string>Description:</string>
       </property>
       <property name="buddy">
        <cstring>m_descriptionTextEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="KTextEdit" name="m_descriptionTextEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>76</height>
        </size>
       </property>
       <property name="tabChangesFocus">
        <bool>true</bool>
       </property>
       <property name="acceptRichText">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="m_verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>13</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QGroupBox" name="m_lessonGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QSplitter" name="m_lessonsSplitter">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="childrenCollapsible">
         <bool>false</bool>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <layout class="QVBoxLayout" name="m_verticalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="m_horizontalLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="m_lessonsLabel">
              <property name="text">
               <string>Lessons:</string>
              </property>
              <property name="buddy">
               <cstring>m_lessonView</cstring>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="m_horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Preferred</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QToolButton" name="m_addLessonButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="toolTip">
               <string>Add new lesson</string>
              </property>
              <property name="icon">
               <iconset theme="list-add"/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="m_removeLessonButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="toolTip">
               <string>Remove lesson</string>
              </property>
              <property name="icon">
               <iconset theme="list-remove"/>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="m_horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>10</width>
                <height>10</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QToolButton" name="m_moveLessonUpButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="toolTip">
               <string>Move lesson up</string>
              </property>
              <property name="icon">
               <iconset theme="arrow-up"/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="m_moveLessonDownButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="toolTip">
               <string>Move lesson down</string>
              </property>
              <property name="icon">
               <iconset theme="arrow-down"/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QListView" name="m_lessonView">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="layoutWidget_2">
         <layout class="QHBoxLayout" name="m_horizontalLayout_3">
          <item>
           <spacer name="m_horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QVBoxLayout" name="m_verticalLayout_3">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <layout class="QFormLayout" name="m_formLayout_2">
              <property name="fieldGrowthPolicy">
               <enum>QFormLayout::ExpandingFieldsGrow</enum>
              </property>
              <property name="horizontalSpacing">
               <number>4</number>
              </property>
              <property name="verticalSpacing">
               <number>4</number>
              </property>
              <item row="0" column="0">
               <widget class="QLabel" name="m_lessonTitleLabel">
                <property name="text">
                 <string>Title:</string>
                </property>
                <property name="buddy">
                 <cstring>m_lessonTitleLineEdit</cstring>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="KLineEdit" name="m_lessonTitleLineEdit">
                <property name="enabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="m_newCharactersLabel">
                <property name="text">
                 <string>New characters:</string>
                </property>
                <property name="buddy">
                 <cstring>m_newCharactersLineEdit</cstring>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="KLineEdit" name="m_newCharactersLineEdit">
                <property name="enabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="m_verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>13</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="LessonTextEditor" name="m_lessonTextEditor" native="true"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KTextEdit</class>
   <extends>QTextEdit</extends>
   <header>ktextedit.h</header>
  </customwidget>
  <customwidget>
   <class>KMessageWidget</class>
   <extends>QFrame</extends>
   <header location="global">kmessagewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KeyboardLayoutComboBox</class>
   <extends>QComboBox</extends>
   <header>editor/keyboardlayoutcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>LessonTextEditor</class>
   <extends>QWidget</extends>
   <header>editor/lessontexteditor.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>m_titleLineEdit</tabstop>
  <tabstop>m_keyboardLayoutComboBox</tabstop>
  <tabstop>m_descriptionTextEdit</tabstop>
  <tabstop>m_lessonView</tabstop>
  <tabstop>m_addLessonButton</tabstop>
  <tabstop>m_removeLessonButton</tabstop>
  <tabstop>m_moveLessonUpButton</tabstop>
  <tabstop>m_moveLessonDownButton</tabstop>
  <tabstop>m_lessonTitleLineEdit</tabstop>
  <tabstop>m_newCharactersLineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
